Dockerfile

```diff
@@ -6,43 +6,41 @@
 # Based on this issue: https://github.com/docker/for-mac/issues/5831
 
 # Install dependencies only when needed
-FROM node:16.3-alpine3.12 AS deps
+FROM node:16-alpine AS deps
```

Comment:
I'm not sure we want to be using the generic image name, it makes the build process more prone to potential issues and less stable.

If a new release of alpine breaks something, we can't easily tell which was the last known working version.

Reply:
Done